broker/lib/fabrik/DirectorInstance.js

```diff
@@ -75,7 +75,11 @@ class DirectorInstance extends BaseInstance {
     };
     return Promise
       .all([
-        this.cloudController.getServiceInstance(this.guid),
+        eventmesh.apiServerClient.getResource({
```

Comment:
When are we getting rid of this class? Per my discussion with Subhankar we are to get rid of this class along with DirectorManager. We seem to be enhancing obsolete classes. What is the plan going forward?

Reply:
Not addressing in this PR.

Reply:
ok, but can you please add a backlog then?